lib/phoenix/transports/websocket.ex

```diff
@@ -2,10 +2,16 @@ defmodule Phoenix.Transports.WebSocket do
   @moduledoc false
   alias Phoenix.Socket.{V1, V2, Transport}
 
+  defmodule ErrorHandler do
+    def handle(conn, :wrong_method), do: Plug.Conn.send_resp(conn, 400, "")
+    def handle(conn, _reason), do: Plug.Conn.send_resp(conn, 403, "")
+  end
```

Comment:
No need to define a separate module for it, just define a `error_handler` function directly in WebSocket. :)